api-get-options.go

```diff
@@ -28,9 +28,8 @@ import (
 // GetObjectOptions are used to specify additional headers or options
 // during GET requests.
 type GetObjectOptions struct {
-	headers map[string]string
-
-	Materials encrypt.Materials
+	headers    map[string]string
+	ServerSide encrypt.ServerSide
```

Comment:
@abperiasamy suggested we should make this more explicit i.e not call it a generic name "ServerSide" instead call it "ServerSideEncryption"

Reply:
Also fine - will change